README.md

```diff
@@ -350,6 +350,40 @@ parser.addPagination('custom-pagination', function (options) {
 * *paginateFn* - Function performs pagination, should return `Promise`.
 * *checkPaginationFn* - Function checks pagination complete, should return `Promise`.
 
+**Example**
+
+Describe new pagination type
+```js
+var previousPageHtml;
+parser.addPagination('clickPerPage', function (options) {
+    var selector = options.scope + ':eq(' + this._currentPage + ')';
+    return this._env
+        .evaluateJs(options.pageScope, this._getPaginatePageHtml)
+        .then(function (html) {
+            previousPageHtml = html;
+            return this._actions.click(selector);
+        }, this);
+}, function (options, timeout) {
+    return this._actions.wait(this._getPaginatePageHtml, function (html) {
+        return html !== null && html !== previousPageHtml;
+    }, [options.pageScope], timeout)
+});
+```
+
+Use it
+```js
+var pagination = {
+  type: 'clickPerPage', // your custom type
+  pageScope: '.page__content',
+  scope: '.page__pagination'
+};
+
+var parser = new Parser({
+    rules: {},
```

Comment:
Again missed.

Take a look on this https://github.com/redco/goose-parser/blob/master/tests/phantom_parser_test.js#L301

For some reason we pass pagination on `Parser` create.
But rules and actions on the `parser.parse()`